docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,244 @@
+and desktops that are joined to Active Directory domains. If you use a commercial edition of 
+Teleport can also mix the automatic discovery provided by Active Directory integration
+with the static host definitions demonstrated in this guide.
 
-For open source Teleport, consider integrating Teleport with Active Directory
-for automatic discovery by reading [Desktop Access with Active Directory](./active-directory.mdx).
-
-Teleport Enterprise users can also mix the Teleport Active Directory integration
-with the static host definitions described below.
-</Admonition>
-
-<Details
-  title="Version warning"
-  opened={true}
-  min="12.0"
->
-Passwordless access for local users is available starting from Teleport `v12`.
-Previous versions of Teleport can implement Windows Access by integrating with
-an [Active Directory domain](./active-directory.mdx).
-</Details>
+For information about integrating Teleport with Active Directory, see [Desktop Access with 
+Active Directory](./active-directory.mdx).
 
 ## Prerequisites
 
+To complete the steps in this guide, verify your environment meets the following requirements:
+
 (!docs/pages/includes/commercial-prereqs-tabs.mdx!)
 
-- A Linux server to run the Teleport Desktop Service on.
+- A Linux server to run the Teleport Windows Desktop Service.
   You can reuse an existing server running any other Teleport instance.
-- A server or virtual machine running a Windows operating system with
-  Remote Desktop enabled and the RDP port available to the Linux server.
+- A physical or virtual machine running Microsoft Windows with
+  Remote Desktop enabled and the RDP port—typically, port 3389—open for inbound connections
+  from the Linux server.
 - (!docs/pages/includes/tctl.mdx!)
 
 ## Step 1/4. Prepare Windows
 
-In this section we'll import the Teleport certificate authority (**CA**) file
-to your Windows system, and prepare it for passwordless access through Teleport.
+Before you can enroll a Microsoft Windows desktop as a resource to be protected by Teleport,
+you need to prepare the Microsoft Windows environment with a certificate from the Teleport 
+certificate authority (CA) and modify the default authentication service for remote desktop
+access.
 
-### Import the Teleport root certificate
+To prepare Microsoft Windows desktop:
 
-Export the Teleport user certificate authority by running the following from
-`cmd.exe` on your Windows system:
+1. Open a Command Prompt (`cmd.exe`) window on the Windows computer you want to enroll.
 
-```code
-$ curl -o teleport.cer https://teleport-proxy.example.com/webapi/auth/export?type=windows
-```
+1. Export the Teleport user certificate authority by running the following command:
+   
+   ```code
+   $ curl -o teleport.cer https://teleport-proxy.example.com/webapi/auth/export?type=windows
+   ```
 
-### Install the Teleport service for Windows
+1. Download the [Teleport Windows Auth
```

Comment:
I'd recommend making it more explicit that this is a link to an executable, rather than to a docs page with information about the program.